tests/fips/openssh/openssh_fips.pm

```diff
@@ -1,33 +1,46 @@
 # SUSE's openssh fips tests
 #
-# Copyright © 2016 SUSE LLC
+# Copyright © 2016 - 2021 SUSE LLC
 #
 # Copying and distribution of this file, with or without modification,
 # are permitted in any medium without royalty provided the copyright
 # notice and this notice are preserved.  This file is offered as-is,
 # without any warranty.
 
 # Package: openssh expect
-# Summary: FIPS: openssh
-#    Involve the existing openssh test case: sshd.pm
 #
-#    Create new case ssh_pubkey.pm to test public key
+# Summary: FIPS: openssh
+#          Involve the existing openssh test case: sshd.pm
+#          Create new case ssh_pubkey.pm to test public key
+#          Create new case openssh_fips.pm to verify that
+#          openssh will refuse to work with any non-approved
+#          algorithm in fips mode, just like blowfish cipher
+#          or MD5 hash.
 #
-#    Create new case openssh_fips.pm to verify that
-#    openssh will refuse to work with any non-approved
-#    algorithm in fips mode, just like blowfish cipher
-#    or MD5 hash.
 # Maintainer: Ben Chou <bchou@suse.com>
-# Tags: tc#1525228
+# Tags: tc#1525228, poo#90458
 
 use base "consoletest";
 use strict;
 use warnings;
 use testapi;
+use utils 'zypper_call';
+use version_utils 'is_sle';
 
 sub run {
     select_console 'root-console';
 
+    zypper_call('info openssh');
+    my $current_ver = script_output("rpm -q --qf '%{version}\n' openssh");
+
+    # openssh update to 8.3 in SLE15 SP3
+    if (is_sle('>=15-sp3') && ($current_ver ge 8.3)) {
+        record_info("openssh version", "Current openssh package version: $current_ver");
+    }
+    else {
+        record_info("openssh version outdate", "openssh version need to be updated over to 8.3+ in SLE15 SP3");
```

Comment:
Ben,
I'd suggest to add some "soft fail here", just recording the info will not give any error/warning, we still need check the result manually.

Reply:
Update the PR. thanks @rfan1